bower.json

```diff
@@ -20,7 +20,7 @@
     "purescript-foreign-object": "^2.0.0",
     "purescript-now": "^4.0.0",
     "purescript-datetime": "^4.0.0",
-    "purescript-psa-utils": "^6.0.0",
+    "purescript-psa-utils": "kl0tl/purescript-psa-utils#censor-user-defined-warnings",
```

Comment:
I’ll point back to a version number once https://github.com/natefaubion/purescript-psa-utils/pull/10 is merged and a new version is released.